### PR TITLE
Improve RRCP test variant 'live' previews

### PR DESCRIPTION
**Pull request by KaliedaRik**
## What does this change?
Existing 'live' preview functionality does not accurately display Banner and Epic test variants to the user - some critical parts of the tests are missing (eg choice cards). Marketing colleagues have to rely on the Web preview (where available) to capture screenshots of their work, which is inefficient and not particularly 'real-time live'.

This PR attempts to fix these issues for the various Epic channel 'live' previews. 

It also removes the 'live' preview panel in the AMP Epic channel, and enables the 'web' preview for that channel

### Screenshots

Epic LIVE preview:

Screenshot 2022-05-17 at 13 34 57

Screenshot 2022-05-17 at 13 42 08

AMP Epic channel page (live preview panel removed):

Screenshot 2022-05-17 at 13 47 17

AMP Epic WEB preview enabled - preview result:

Screenshot 2022-05-17 at 13 54 45

### CODE REVIEW

File: public/src/components/channelManagement/epicTests/epicVariantPreview.tsx [hunk 1]
```diff
@@ -2,16 +2,102 @@ import React from 'react';
...
+}
+
+interface ChoiceCardVariant {
+  name: string;
+  amounts: {
+    [index: string]: ChoiceCardAmounts;
+  };
+}
+
+interface RegionalChoiceCard {
```
Comment (tomrf1) [inline public/src/components/channelManagement/epicTests/epicVariantPreview.tsx:29]: Can we use the same model as SDC? https://github.com/guardian/support-dotcom-components/blob/main/packages/shared/src/types/abTests/epic.ts#L77
Ideally these types would be published as a library, but that's for another day...

File: public/src/components/channelManagement/helpers/shared.ts [hunk 2]
```diff
@@ -45,6 +45,7 @@ export interface EpicEditorConfig {
   allowVariantTicker: boolean;
   allowVariantChoiceCards: boolean;
   allowVariantSignInLink: boolean;
+  allowVariantPreview: boolean;
```
Comment (tomrf1) [inline public/src/components/channelManagement/helpers/shared.ts:48]: 👍

File: public/src/components/channelManagement/epicTests/epicVariantPreview.tsx [hunk 3]
```diff
@@ -35,11 +121,49 @@ const buildProps = (variant: EpicVariant): EpicProps => ({
...
+      reminderLabel: 'October',
+    },
+
     separateArticleCount: variant.separateArticleCount,
-    showSignInLink: variant.showSignInLink,
+
+    // There's some weird stuff going on around the sign-in link
+    // - we have an 'enable sign-in link' checkbox in RRCP
+    // - but no indication that the checkbox setting gets set in variant data
+    // - for now, always show the link for the purposes of LIVE preview
```
Comment (tomrf1) [inline public/src/components/channelManagement/epicTests/epicVariantPreview.tsx:138]: I'm not sure what you mean - this seems to work fine -
Screen Shot 2022-05-18 at 08 22 35

This should be `showSignInLink: variant.showSignInLink`

File: public/src/components/channelManagement/epicTests/epicVariantPreview.tsx (hunk 1 above)
Comment (KaliedaRik) [reply public/src/components/channelManagement/epicTests/epicVariantPreview.tsx:29]: Done

File: public/src/components/channelManagement/epicTests/epicVariantPreview.tsx (hunk 3 above)
Comment (KaliedaRik) [reply public/src/components/channelManagement/epicTests/epicVariantPreview.tsx:138]: Assuming Marketing colleagues have chosen not to set the attribute for any of their tests currently showing in RRCP LIVE (which led to my misunderstanding), code updated as suggested

**Review by tomrf1 — APPROVED** [review]
great work

**Pull request merged**